examples/node-stubbed/index.js

```diff
@@ -24,7 +24,7 @@ const msleep = n => {
 };
 
 // Set up our file system (NOTE: We could use node's fs normally for this case)
-const wasmerFs = new WasmerFileSystem();
+const wasmerFs = new WasmerFileSystem.default();
```

Comment:
?

Reply:
This is because the file system has a default export, but in node this is an object, so you gotta use default to access efault export.

`const { WasmFs } = require('wasmfs')` works as well 😄